### PR TITLE
snscrape script

### PR DESCRIPTION
**Pull request by ashioyajotham**
This script utilizes snscrape and checks over three key words in the protests to scrape 1000 rows

### CODE REVIEW

File: data_collection/sns_scrape.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# We are going to scrape Twitter using snscrape
```
Comment (VictorOmondi1997) [inline data_collection/sns_scrape.py:1]: To make sure the lint is passing, the python file line 1 needs to be a docstring to explain what the .py file is about.
```suggestion
"""We are going to scrape Twitter using snscrape"""
```

File: data_collection/sns_scrape.py
Comment (VictorOmondi1997) [inline data_collection/sns_scrape.py:1]: Thank you @ashioyajotham for exploring snscrape library. This looks great, but there are some few things that needs to be addressed before we merge it to main:
image

When experiencing this 👇🏿  just make sure the library is part of our `requirements.txt`
```
data_collection/sns_scrape.py:3:0: E0401: Unable to import 'pandas' (import-error)
data_collection/sns_scrape.py:4:0: E0401: Unable to import 'numpy' (import-error)
```

File: data_collection/sns_scrape.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+import pandas as pd
+import numpy as np
+import snscrape.modules.twitter as sntwitter
+import csv
+import datetime
```
Comment (VictorOmondi1997) [inline data_collection/sns_scrape.py:7]: There are some libraries that you've imported that have not been used in this script, those are `numpy` and `csv`. Our lint tool will show you some of this.
```suggestion
import pandas as pd
import snscrape.modules.twitter as sntwitter
import datetime
```

**Review by VictorOmondi1997 — CHANGES_REQUESTED** [review]

**Review by 3liud — APPROVED** [review]

**Review by Stephen-Kamau — APPROVED** [review]
good

**Pull request merged**